From 9b991967922faf0c95455a3f77e3dbdd88ed768d Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:38:42 -0500
Subject: [drm] ast: AST2000 cannot be detected correctly

Message-id: <1418852380-13061-32-git-send-email-rclark@redhat.com>
Patchwork-id: 102173
O-Subject: [RHEL7 drm 31/89] drm/ast: AST2000 cannot be detected correctly
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: since v3.16.4

commit c84162d160a44cdd29689d536adc0005e7b740c6

Author:     Y.C. Chen <yc_chen@aspeedtech.com>
AuthorDate: Wed Sep 10 12:07:54 2014 +0800
Commit:     Rob Clark <rclark@redhat.com>
CommitDate: Wed Dec 10 17:16:29 2014 -0500

    drm/ast: AST2000 cannot be detected correctly

    commit 83502a5d34386f7c6973bc70e1c423f55f5a2e3a upstream.

    Type error and cause AST2000 cannot be detected correctly

    Signed-off-by: Y.C. Chen <yc_chen@aspeedtech.com>
    Reviewed-by: Egbert Eich <eich@suse.de>
    Signed-off-by: Dave Airlie <airlied@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/ast/ast_main.c b/drivers/gpu/drm/ast/ast_main.c
index b027e33..b792194 100644
--- a/drivers/gpu/drm/ast/ast_main.c
+++ b/drivers/gpu/drm/ast/ast_main.c
@@ -105,7 +105,7 @@ static int ast_detect_chip(struct drm_device *dev)
    }
    ast->vga2_clone = false;
   } else {
-   ast->chip = 2000;
+   ast->chip = AST2000;
    DRM_INFO("AST 2000 detected\n");
   }
  }
-- 
1.7.1